ci/Dockerfile.ci-utils

```diff
@@ -1,7 +1,20 @@
 FROM {{ base_image.image }}
 
+# Installing kubectl on Ubuntu
+# https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-using-native-package-management
```

Comment:
Why this change?  The base image already has kubectl.

Reply:
I didn't see the kubectl in the base image. Fixed.